product_docs/docs/pgd/3.6/release-notes.mdx

```diff
@@ -1,7 +1,23 @@
 ---
-title: Release Notes
+title: BDR Release Notes
 ---
 
+## BDR 3.6.33 (2022 Nov 16)
+
+This is a maintenance release for BDR 3.6 which includes
+fixes for issues identified previously.
+
+### Resolved Issues
+
+- Fix bdr.alter_table_conflict_detection() (BDR-2650)  
+  Correct the function to use the 'detector_args' argument.
+
+### Upgrades
+
+This release supports upgrading from the following versions of BDR:
+
+ - 3.6.20 and higher
+
 ## BDR 3.6.2 (2022 Aug 23)
```

Comment:
I see there is an issue with the previous release, it should be `3.6.32`, not `3.6.2`